packages/graphback-datasync/src/deltaHelper.ts

```diff
@@ -0,0 +1,46 @@
+import { ModelDefinition, getTableName } from '@graphback/core';
+import { Db, Collection, ObjectId } from "mongodb";
+import { DataSyncFieldNames } from './util';
+
+export function getDeltaTableName(tableName: string) {
+  return `${tableName}Delta`
+}
+
+/**
+ * M
+ */
+export class MongoDeltaHelper {
```

Comment:
> a helper class is used to assist in providing some functionality, which isn't the main goal of the application or class in which it is used. 

I think the name `MongoDeltaHelper` is too generic/non-specific. It is in fact a very integral class providing a very specific purpose. I see a helper class as being a collection of methods which do not necessarily work together or have common purpose.